exalearn-ui/src/app/pages/main-page/main-page.component.scss

```diff
@@ -18,7 +18,10 @@
 }
 
 ::ng-deep div.title {
-	font-size: $font-size-h3;
-	margin: 30px 50px 40px;
-	text-align: left;
+	font-size: $font-size-h5;
+	margin: 30px 0 40px;
+	text-align: center;
+	@media (max-width: 1200px) {
+		font-size: calc((100vw - 17.5rem) / (75 - 17.5) * (2 - 1) + 1rem);
```

Comment:
It looks so complicated just for title font-size) Why do we need such a huge calc here?